### DISCUSSION TITLE
COCO评价指标大中小物体的判别area是相对于原图么？

### DISCUSSION THREAD

**Discussion by Yuanyang-Zhu** [open-mmlab/mmyolo#798] [Q&A]
COCO指标中有APs是小于32×32的大小的bbox，然后APl是大于96×96的bbox。这里的32×32是相对于原图的大小，还是相对于网络输入（比如resize到640×640）的大小？从指标代码看，好像直接相对于原图的？

**Comment by josh3255** [comment]
performance indicator is the size of the source image.

**Reply by Yuanyang-Zhu** [reply]
I learned a lot. Thank you very much!
